src/components/Nav.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Bird } from 'phosphor-react'
+
+export const Nav = () => {
```

Comment:
Since this is the only component in the file export it as default.
Also do this for the other components.